chart/linkerd-buoyant/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: linkerd-buoyant
+version: 0.2.0
```

Comment:
can the version be templatized? currently we don't hard-code the `linkerd-buoyant` version anywhere in the repo.